examples/pxScene2d/src/pxContextGL.cpp

```diff
@@ -27,6 +27,8 @@
 #include "rtScript.h"
 #include "rtSettings.h"
 
+#include "pxShaderResource.h"
```

Comment:
Can you please remove the dependency of pxShaderResource (which depends on pxResource) from pxContext?

Reply:
Done